APP-MANAGER

```diff
@@ -2,6 +2,10 @@
 
 AMVERSION="6.6.4"
 
+# Determine main repository and branch
```

Comment:
because $AMREPO is in all scripts, in my tests I do manually on my PC
```
#!/usr/bin/env bash

AMVERSION="6.6.4"

# Determine main repository and branch
AMREPO="https://raw.githubusercontent.com/ivan-hc/AM/dev"
AMBRANCH=$(basename "$AMREPO")
```
so I don't need to add "newrepo", its quickier for me

Reply:
I also can do
```
#!/usr/bin/env bash

AMVERSION="6.6.4"

# Determine main repository and branch
AMBRANCH="dev"
AMREPO="https://raw.githubusercontent.com/ivan-hc/AM/$AMBRANCH"
```
@nazdridoy is this OK?

Reply:
@nazdridoy https://github.com/ivan-hc/AM/pull/533/commits/e1f367e4fc9b2b661133a5475de5f7d7c2dbc4ea

Reply:
this order is needed for our work, while testing

Reply:
@nazdridoy download manually both APP-MANAGER and sync.am and put them where they should be

then, edit APP-MANAGER and change
```
AMBRANCH="main"
```
to
```
AMBRANCH="dev"
```
and then run
```
am -s
```
you will get the modules from this repo (and the CLI as well, so you have to change "main" to "dev" again)

Reply:
I can als create a "devmode" option to let "dev" automatically enabled